src/seeds/users/001_users.ts

```diff
@@ -0,0 +1,82 @@
+import * as bcrypt from 'bcrypt';
+import { randomUUID } from 'crypto';
+import { Knex } from 'knex';
+
+const initialUsers = [
+  {
+    nickname: 'admin',
+    given_name: 'Admin',
+    family_name: 'User',
+    privilege: 'admin',
+    resource: '*',
+    scope: '*',
```

Comment:
I don't think these 2 are used.

Reply:
I was copying what came in through the initial admin registration 😜